sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3402,6 +3402,53 @@ object GpuOverrides extends Logging {
           GpuVarianceSamp(childExprs.head, !legacyStatisticalAggregate)
         }
       }),
+    expr[Percentile](
+      "Aggregation computing exact percentile",
+      ExprChecks.reductionAndGroupByAgg(
+        // The output can be a single number or array depending on whether percentiles param
+        // is a single number or an array.
+        TypeSig.gpuNumeric + TypeSig.ARRAY.nested(TypeSig.gpuNumeric),
+        TypeSig.cpuNumeric + TypeSig.ARRAY.nested(TypeSig.cpuNumeric),
```

Comment:
Technically the CPU supports numerics, but also ANSI interval types. YearMonthIntervalType and DayTimeIntervalType. It is a bit of a nit here because the version of Spark we are generating the docs for does not support these types, but it would be nice to at least have a comment here about it that when 3.2.0 is the minimum version we should update this.